components/grid/blender/_utils.js

```diff
@@ -0,0 +1,6 @@
+export const rowMap = {
+	columns: 12,
+	gap: [12, 24],
+};
+
+// export const colMap = {};
```

Comment:
Can clean this...

Reply:
Yeah sweet no probs

Could also get the `columns` and `gap` values above by exporting the defaultValues out of `Grid.js`. Is that worth considering?

Reply:
^ like this https://github.com/WestpacGEL/GEL/commit/ba2682bbcd35b2c9dc7770fb6985732fe3e8a335 ?